3-networks/shared.auto.example.tfvars

```diff
@@ -14,4 +14,6 @@
  * limitations under the License.
  */
 
+// List of IPv4 address of target name servers for the forwarding zone configuration.
+// Cloud DNS will select the best available name server.
```

Comment:
Can you elaborate what you mean here in terms of Cloud DNS selecting the best available name server?

Reply:
Hi @rjerrems this was an extract from the dns.googleapis.com documentation for forwardingConfig.targetNameServers 
https://cloud.google.com/config-connector/docs/reference/resource-docs/dns/dnsmanagedzone#spec

I replaced it with a link to the [Cloud DNS forwarding zones](https://cloud.google.com/dns/docs/overview#dns-forwarding-zones) documentation that will better explain what these servers are used for.